diff --git a/node_modules/react-native-material-dropdown/src/components/dropdown/index.js b/node_modules/react-native-material-dropdown/src/components/dropdown/index.js
index b1e65a1..33e439a 100644
--- a/node_modules/react-native-material-dropdown/src/components/dropdown/index.js
+++ b/node_modules/react-native-material-dropdown/src/components/dropdown/index.js
@@ -75,7 +75,7 @@ export default class Dropdown extends PureComponent {
       'landscape-right',
     ],
 
-    useNativeDriver: false,
+    useNativeDriver: true,
   };
 
   static propTypes = {
@@ -299,7 +299,7 @@ export default class Dropdown extends PureComponent {
             .timing(opacity, {
               duration: animationDuration,
               toValue: 1,
-              useNativeDriver,
+              useNativeDriver: true,
             })
             .start(() => {
               if (this.mounted && 'ios' === Platform.OS) {
@@ -323,7 +323,7 @@ export default class Dropdown extends PureComponent {
       .timing(opacity, {
         duration: animationDuration,
         toValue: 0,
-        useNativeDriver,
+        useNativeDriver: true,
       })
       .start(() => {
         this.focused = false;
